### PR TITLE
Compress read char 261 data exception

### PR DESCRIPTION
**Pull request by bpcvdhelm**
This pull request correctly implements a data exception when reading character 261 during compression. The current implementation is wrong! it counts the number of tested children in the character entry and sibling descriptor. This wrong data exception implementation never fires of on valid compression tables. But when invalid tables are used the results are (as stated in the manual ;-) unpredictable.

Short version: There is no hurry to implement this change!

### CODE REVIEW

**Comment by rbowler** [comment]
These changes have been included in release 3.12.
